www/js/controllers/reality-check.controller.js

```diff
@@ -0,0 +1,132 @@
+angular
+	.module('binary')
+	.controller('RealityCheckController',
+		function($scope, $rootScope, $state, $timeout, $interval, websocketService, appStateService, accountService, $ionicPopup, alertService, localStorageService) {
+			var show,
+				myInterval,
+				i = 0,
+				_landing_company_name_s,
+				myflag = 0,
+				myRequestFlag = 0;
+
+			var auth = function auth() {
+				if (!appStateService.isLoggedin) {
+					$timeout(auth, 500);
+				} else {
+					$scope.init();
+				}
+			};
+
+			$scope.init = function() {
+				$scope.$on('authorize', function(e, authorize) {
+					if (myflag == 0) {
+						myflag = 1;
+						_landing_company_name_s = authorize.landing_company_name;
+						$scope.hasRealityCheck();
+					}
+				});
+			};
+
+
+			$scope.hasRealityCheck = function() {
+				if (localStorage._interval) {
```

Comment:
you should show the first popup every time that the app is opened. so you should move your interval to sessionStorage instead of localStorage